src/buildWrapperMap.ts

```diff
@@ -0,0 +1,57 @@
+import packageJson from '../package.json';
+import tokenJson from './tokens.json';
+import { WrapperPair, WrapperMap, TokenConfig } from './types';
+
+function getAddressKeyedTokenMap(tokenJson: TokenConfig[]): {
+    [key: string]: TokenConfig;
```

Comment:
Worth defining a type for the KeyedTokenMap rather than defining it inline in multiple places I think